### ISSUE TITLE
Implement search portal changes

### ISSUE THREAD

**Issue by ascourtas** [MLMI2-CSSI/foundry#352]
_As a Dataset Consumer, I want to be able to easily find Foundry and MDF datasets. It's best if there is one, easy-to-navigate search portal_

One Search would be built using the [ALCF current search page](https://acdc.alcf.anl.gov/mdf/?q=*&filter-match-any.mdf.resource_type=dataset) ([repo here](https://github.com/globusonline/django-alcf-data-portal))--redesign it. There are filters already in place. Use what you think are useful in your design.

We would like one search page that can be filtered by MDF, Foundry, or Garden.

The tasks below may be broken up into smaller issues for ease of completion.

```[tasklist]
### Tasks
- [ ] Update design: incorporate all UI/UX changes outlined in the previous issues in this milestone
- [ ] Implement auto-filters: when a user clicks on "Find Datasets" or "Browse Data" in the [MDF](https://www.materialsdatafacility.org/) or [Foundry](https://foundry-ml.org/#/) website, they are taken to the dataset portal ([at this domain](https://acdc.alcf.anl.gov/mdf/?q=*)) and the filter for MDF or Foundry is auto-applied
```

# Assumptions:
1. The UI/UX redesigns have already been completed
2. We are still using the domain https://acdc.alcf.anl.gov/mdf/?q=* as our basis for search
3. We are totally redesigning the look and feel of 2), and are using what's there as a starting point

# Acceptance Criteria
1. Given the UI/UX designs, the implementation fully realizes them and gives a clean, polished, intuitive finish
2. Given a user is navigating the dataset search portal, it should feel like a cohesive website with consistent styling and formats

**Comment by muchanem** [comment]
One question here: how heavily can/should I pull from garden styles/design language for this page. On the one hand, it does sort of "unify" the accelerate team products and I imagine many of the intial users of garden will be MDF/Foundry users, so it can be helpful to have one single look/feel. On the other hand, we definitely don't want to confuse people with styles implying that things are the same when they're not. There's also definitely levels to this. There's a difference between using the same fonts and using the exact same color scheme (I feel like using the green colors of garden here wouldn't be the right choice, just because the green is an intentional choice for gardens)



Sorry that this is already so long, but the place where I land on this is: MDF/Foundry colors (keeping blue hyperlinks, lavendar accents, etc) with the look and feel of Garden (fonts/general component styling, etc).

**Comment by kjschmidt913** [comment]
Mark has access!!!! 

This task is now blocked by migrating the code from the ALCP repo to the MDF repo. We had to switch to a new plan

**Comment by kjschmidt913** [comment]
This changed into just being the MDF search portal

**Comment by kjschmidt913** [comment]
Oh I see there is already an MDF issue for this on the community board. Closing this version
